src/ServiceControl.Monitoring/Infrastructure/LoggingConfigurator.cs

```diff
@@ -16,96 +12,54 @@
-            nlogConfig.AddTarget("null", nullTarget);
-
-            //Suppress NSB license logging since this will have it's own
-            nlogConfig.LoggingRules.Add(new LoggingRule("NServiceBus.LicenseManager", LogLevel.Info, nullTarget) { Final = true });
-
             // Always want to see license logging regardless of default logging level
             nlogConfig.LoggingRules.Add(new LoggingRule("ServiceControl.Monitoring.Licensing.*", LogLevel.Info, fileTarget));
-            nlogConfig.LoggingRules.Add(new LoggingRule("ServiceControl.Monitoring.Licensing.*", LogLevel.Info, consoleTarget)
-            {
-                Final = true
-            });
+            nlogConfig.LoggingRules.Add(new LoggingRule("ServiceControl.Monitoring.Licensing.*", LogLevel.Info, consoleTarget));
 
             // Defaults
-            nlogConfig.LoggingRules.Add(new LoggingRule("*", settings.LogLevel, fileTarget));
-            nlogConfig.LoggingRules.Add(new LoggingRule("*", settings.LogLevel < LogLevel.Info ? settings.LogLevel : LogLevel.Info, consoleTarget));
-
-            if (!logToConsole)
-            {
-                foreach (var rule in nlogConfig.LoggingRules.Where(p => p.Targets.Contains(consoleTarget)).ToList())
-                {
-                    nlogConfig.LoggingRules.Remove(rule);
-                }
-            }
+            nlogConfig.LoggingRules.Add(new LoggingRule("*", loggingSettings.LogLevel, fileTarget));
+            nlogConfig.LoggingRules.Add(new LoggingRule("*", loggingSettings.LogLevel < LogLevel.Info ? loggingSettings.LogLevel : LogLevel.Info, consoleTarget));
 
             NLog.LogManager.Configuration = nlogConfig;
 
             LogManager.UseFactory(new ExtensionsLoggerFactory(new NLogLoggerFactory()));
 
             var logger = LogManager.GetLogger("LoggingConfiguration");
-            var logEventInfo = new LogEventInfo
-            {
-                TimeStamp = DateTime.UtcNow
-            };
-            logger.InfoFormat("Logging to {0} with LogLevel '{1}'", fileTarget.FileName.Render(logEventInfo), settings.LogLevel.Name);
+            var logEventInfo = new LogEventInfo { TimeStamp = DateTime.UtcNow };
+            logger.InfoFormat("Logging to {0} with LogLevel '{1}'", fileTarget.FileName.Render(logEventInfo), loggingSettings.LogLevel.Name);
         }
 
-        public static LogLevel InitializeLevel()
-        {
-            var level = LogLevel.Warn;
-            try
-            {
-                level = LogLevel.FromString(SettingsReader.Read(Settings.SettingsRootNamespace, LogLevelKey, LogLevel.Warn.Name));
-            }
-            catch
-            {
-                InternalLogger.Warn($"Failed to parse {LogLevelKey} setting. Defaulting to Warn.");
-            }
-
-            return level;
-        }
-
-        const long MegaByte = 1024 * 1024;
-
-        const string LogLevelKey = "LogLevel";
+        const long megaByte = 1024 * 1024;
```

Comment:
> Use PascalCase for constant names, both fields and local constants.
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

That would be my preference but I realize we have no enforcement so it is just my opinion